Makefile

```diff
@@ -866,6 +866,17 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint-build:
```

Comment:
That's right! I'm going to update its behaviour as like how `make coverage` work. It will have `mdlint-clean`.